### ISSUE TITLE
Campaign page in draft has pink block over hero image

### ISSUE THREAD

**Issue by lgontard** [ytgov/yukon-ca#761]
When a campaign page is in draft, the pink background that indicates a page is in draft appears as a block over the hero image. The pink draft status should appear behind the hero image

> [!IMPORTANT]
> **Work-around:** Publish page and the pink block disappears.

![image](https://github.com/user-attachments/assets/91c38ed3-e93b-4ba1-a0b4-f6d2909ee78e)

**Comment by leeomara** [comment]
Yes, the goal is to clearly distinguish published from non-published content. I should have written translucent (the CSS example I tested was 50% opacity). I'll see if I can't get a screenshot.

**Comment by w3creatives** [comment]
Please check PR - #807

**Comment by leeomara** [comment]
Testing in UAT, this looks good.

![Screenshot 2025-01-03 at 2 55 21 PM](https://github.com/user-attachments/assets/5f5a8f3a-f9df-4732-bb71-059795db60fe)

**Comment by leeomara** [comment]
After rolling out #815, this appears fixed in prod.
